### PR TITLE
Included event sorting in relevant Controller

### PR DESCRIPTION
**Pull request by x3N1GM4x**
Added a couple of sorts to the OPController so that Event Kits are return by Expiry Date and then event type.

### CODE REVIEW

File: app/Http/Controllers/Web/OPController.php [hunk 1]
```diff
@@ -26,6 +26,8 @@ public function indexForStores(Request $request)
             ->where('active', 1)
             ->whereNull('deleted_at')
             ->select('name', 'slug', 'type', 'price', 'listing_image', 'expires_on')
+            ->orderBy('expires_on','desc')
+            ->orderBy('type','desc')
```
[the remark this answers is not in the document]
Comment (x3N1GM4x) [reply app/Http/Controllers/Web/OPController.php]: Doesn't it order in the same order that they are defined in your snippet above? That is the behaviour I'm seeing, although admittedly I didn't create an event kit for every single type to check 100%...

[the remark this answers is not in the document]
Comment (Absotively) [reply app/Http/Controllers/Web/OPController.php]: I suspect it's likely to be alphabetical.

[the remark this answers is not in the document]
Comment (x3N1GM4x) [reply app/Http/Controllers/Web/OPController.php]: Ah, ok, it's just coincidental then that in my test I had a Regional with a newer Expires On date, then an SC and GNK with the same Expires On, which got correctly sorted, I see. I'll test properly to confirm and then re-think. Could add a `tournament_level` or something so we can sort them by descending "prestige" of tournament when they share an Expires On date (which to be fair should be rare).

[the remark this answers is not in the document]
Comment (x3N1GM4x) [reply app/Http/Controllers/Web/OPController.php]: But actually, it probably makes as much sense to sort on Expires On and then Created At and then if necessary fiddle the Created At dates on the events in production to fit the order I want (which is what I was originally going to do, lol).

**Comment by x3N1GM4x** [comment]
Deployed to production.

**Pull request merged**